pkg/util/tracing/tracingpb/recording.go

```diff
@@ -219,6 +219,25 @@ func (r Recording) FindLogMessage(pattern string) (string, bool) {
 	return "", false
 }
 
+// FindLastLogMessage returns the last log message in the recording that matches
+// the given regexp. The bool return value is true if such a message is found.
+//
+// This method strips the redaction markers from all the log messages, which is
+// pretty inefficient.
+func (r Recording) FindLastLogMessage(pattern string) (string, bool) {
```

Comment:
With only one usage, I think just make this a private function in `kvprober.go` for now. If KV folks think it should be here, they will ask you to move it, but then you'll need more tests, etc.

Reply:
I have moved it to `kvprober.go` for now.